portal/portal-charon/charon/src/webapp/scripts/jumptotop.js

```diff
@@ -25,6 +30,9 @@
 
     // just in case we"re scrolled, assume it is
     this.onScroll();
+
+    window.console && console.debug('JumpToTop was initialized');
```

Comment:
I don't think there's any point doing the window.console check here.

According to ...
https://caniuse.com/#feat=console-basic
... IE10 and upwards work whether the dev tools are open or not. Maybe you don't need the window.console test any longer?

Reply:
Yeah, I could just remove it, I think I left it in there as I wanted to make sure it was actually loading and forgot to clean it up. Though having these statements in at debug can be helpful.